packages/components/src/Page/Page.tsx

```diff
@@ -32,21 +34,51 @@ interface PageProps {
    * @default standard
    */
   readonly width?: "fill" | "standard" | "narrow";
+
+  /**
+   * Page title primary action button settings.
+   */
+  readonly primaryAction?: ButtonProps;
+
+  /**
+   * Page title secondary action button settings.
+   */
+  readonly secondaryAction?: ButtonProps;
+
+  /**
+   * Page title Action menu.
+   */
+  readonly moreActionsMenu?: SectionProps[];
 }
 
 export function Page({
   title,
   intro,
   children,
   width = "standard",
+  primaryAction,
+  secondaryAction,
+  moreActionsMenu = [],
 }: PageProps) {
-  const className = classnames(styles.page, styles[width]);
+  const pageStyles = classnames(styles.page, styles[width]);
+
+  const showMenu = moreActionsMenu.length > 0;
+  const showActionGroup = showMenu || primaryAction;
 
   return (
-    <div className={className}>
+    <div className={pageStyles}>
       <Content>
         <Content spacing="large">
-          <Heading level={1}>{title}</Heading>
+          <div className={styles.titleBar}>
+            <Heading level={1}>{title}</Heading>
+            {showActionGroup && (
```

Comment:
Do we still need this condition here?

Reply:
Yup. We still could have a button or a menu.

Reply:
Sorry, what I was meaning is that we're doing these checks lower down and if those are all false the CSS should collapse everything.

Reply:
Keeping it to avoid adding an empty div into the page.